src/core.c/core_epilogue.pm6

```diff
@@ -135,6 +135,30 @@ BEGIN .^compose for
 
 BEGIN Metamodel::ClassHOW.exclude_parent(Mu);
 
+# This code can be removed once RakuAST is stable and
+#   use experimental :rakuast;
+# is no longer necessary to be able to access the RakuAST classes
+# and their functionality.
+{
+    my Mu $ctx := nqp::getattr(CORE::, PseudoStash, '$!ctx');
+    until nqp::isnull($ctx) {
+        my $pad := nqp::ctxlexpad($ctx);
+        if nqp::existskey($pad, 'CORE-SETTING-REV') {
+            nqp::bindcurhllsym('RakuAST',RakuAST);
+            nqp::bindkey($pad,'RakuAST',Nil);
+            last;
+        }
+        $ctx := nqp::ctxouterskipthunks($ctx);
+    }
+}
```

Comment:
Why haven't you taken my approach with modified RakuAST compiler? The key difference is that any error messages, produced with `Nil` are going to be different from other missing symbol cases and thus will be confusing.

Reply:
I tried that approach, but in the end it breaks down on `sub dd` referencing `RakuAST::Node` in a hard way, but that could be (and is) fixed.

But also `sub EVAL` is referencing `RakuAST::Node` the hard way in a signature.

Also: apparently, `hllsym` values apparently are not cached in the precomps, so it needs to be set at runtime.  At least, I couldn't get it to work.

Reply:
This is not a BEGIN block?  It's an ordinary block that runs at runtime?  The block is just the keep any definitions outside of the setting.

Reply:
Actually, since `RakuAST` is defined by BOOTSTRAP, this is where it is the best to have the binding. It would also be the simplest logic because it's just the bind itself, no outers scanning is needed. The only "but" is that it has to be at the end of gen/ast.nqp, i.e. must be produced by RakuAST compiler. A very little tiny "but", I'd say.

Reply:
Ah, good point...  I'll look at it tomorrow...

Meanwhile, there's something really screwy going in with this patch, so I think I want to move RakuAST::Deparse back into core (as that's where it's going to live in the future anyway).  But now, I'm too tired and not sober enough  :-)